andela/trime
# Filter trimes

### What does this PR do?
Adds filters for the **All Tracks** list on the project page.

### Description of Task to be completed?
Add filters for the **All Tracks** list on the project page. The filters add are:
* Time tracked
  - Min (in minutes)
  - Max (in minutes)
* Description
* Date logged
  - From - The earliest date to include
  - To  - The latest date to include

### How should this be manually tested?
Go to the project page of a project with many **Trimes**. You should see input fields above the the **All Tracks**. If you enter some input in those fields, you should see only **Trimes** that meet the conditions in your input.

### What are the relevant pivotal tracker stories?
[Project Page - User should be able to filter his/her logged time by description, tags, date, hour range](https://www.pivotaltracker.com/story/show/120097411)

#### Screenshots
<img width="1680" alt="screen shot 2016-07-05 at 14 26 42" src="https://cloud.githubusercontent.com/assets/18402277/16583535/d4611684-42be-11e6-90e5-a32dbf887246.png">

All **Trimes** in a project unfiltered

<img width="1680" alt="screen shot 2016-07-05 at 15 07 38" src="https://cloud.githubusercontent.com/assets/18402277/16584071/4f8e40cc-42c2-11e6-9226-f95b20b2a087.png">

**Trimes** in a project filtered but still more than a single page

<img width="1680" alt="screen shot 2016-07-05 at 15 08 00" src="https://cloud.githubusercontent.com/assets/18402277/16584079/57e88728-42c2-11e6-92ca-17f968048755.png">

**Trimes** in a project filtered but every filter.
